src/widgets/Pagination.js

```diff
@@ -0,0 +1,34 @@
+import {
+  pagination as paginationWidget,
+  panel,
+} from 'instantsearch.js/es/widgets';
+
+const paginationWithMultiplePages = panel({
+  hidden({ results }) {
+    return results.nbPages <= 1;
+  },
+})(paginationWidget);
+
+export const pagination = paginationWithMultiplePages({
+  container: '[data-widget="pagination"]',
+  scrollTo: '.container',
+  padding: 2,
+  showFirst: false,
+  showLast: false,
+  templates: {
+    previous: `
+<svg xmlns="http://www.w3.org/2000/svg" width="10" height="10" viewBox="0 0 10 10">
```

Comment:
I wonder if these options couldn't be their default instead to make the example simpler

Reply:
I took the ones from the e-commerce example.